pkg/cache/manager.go

```diff
@@ -106,3 +111,28 @@ func (m *Manager) AddSnapshot(imageID string, blobs []string) error {
 func (m *Manager) DelSnapshot(imageID string) error {
 	return m.db.DelSnapshot(imageID)
 }
+
+// Report each blob disk usage
+func (m *Manager) CacheUsage(ctx context.Context, blobID string) (snapshots.Usage, error) {
+	var usage snapshots.Usage
+
+	blobCachePath := path.Join(m.cacheDir, blobID)
+	blobChunkMap := path.Join(m.cacheDir, blobID+chunkMapFileSuffix)
+	blobMeta := path.Join(m.cacheDir, blobID+metaFileSuffix)
+
+	stuffs := []string{blobCachePath, blobChunkMap, blobMeta}
+
+	for _, f := range stuffs {
+		du, err := fs.DiskUsage(ctx, f)
+		if err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				log.L.Warnf("Cache %s does not exist", f)
```

Comment:
Maybe we can ignore the error directly, otherwise many warnings will be printed?

Reply:
Agree, like bootstrap layer and container rw layer do not have these files, maybe change to Debugf or ignore?

Reply:
Let's change it to a debug message

Reply:
fixed